source/nordic_sdk/components/ble/common/ble_date_time.h

```diff
@@ -30,8 +30,8 @@
  *
  */
 
-/* Attention! 

```

Comment:
On the next line, you will see that I have replaced the character `’` by `'`. 
It was the cause of some issue during the synchronization with mercurial.